### DISCUSSION TITLE
How to get the longest codepoint component in the GSUB table ?

### DISCUSSION THREAD

**Discussion by LukiYLS** [harfbuzz/harfbuzz#3849] [Q&A] [answered]
Such as 👨‍👩‍👧‍👦
codepoint count is 7
I want get longest codepoint set in the GSUB table

**Comment by behdad** [comment, marked as answer]
This is not something that HarfBuzz supports. You would need to get it using a library like the Python FontTools. There's code similar to that (but for the backtrack/lookahead part of the rules) here:

  https://github.com/fonttools/fonttools/blob/main/Lib/fontTools/otlLib/maxContextCalc.py
